src/styles/blocks/card.css

```diff
@@ -0,0 +1,70 @@
+.card {
+  max-width: var(--card-width);
+  max-height: var(--card-height);
```

Comment:
no need to indicate it explicitly, all the content inside `card` dictates what height it'll have